app/views/players/edit.html.slim

```diff
@@ -2,16 +2,7 @@ h1.title
   | EDIT PLAYER
 .form
   = form_for :player, url: player_path(@player), method: :patch do |f|
```

Comment:
@weiqingtoh The `form_for` for edit is 
``` 
= form_for :player, url: player_path(@player), method: :patch do |f|
```
and the `form_for` for new is 
```
= form_for :player, url: players_path do |f|
```
So I'm not so sure if i can pass the `@player` variable to both.